### ISSUE TITLE
Required Fields Missing Alerts   No Indication or Alert for Required Form Fields (Android + iOS) 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5845]
**_Specs: Android
Screen Readers Used: Talkback
Browsers Tested: Google Chrome_**

**_Specs: Apple iOS
Screen Readers Used: Voiceover
Browsers Tested: Safari_**

**Observation:** 
In the form, options “Are you applying from within Canada? “Yes”, “No (leads to Exit Page)” must be selected to proceed to the next page. However: 

There is no visible or screen-reader-accessible indication that these fields are mandatory. 

If a user skips them and taps “Continue”, the form does not provide an error message or feedback. 

As a result, screen reader users (e.g., TalkBack users) may be unaware that input is required. 

**_Compounded Severity: High_** 

**Recommendation:** 
1. Users are informed that the options are mandatory. 

If a user skips the required field and taps “Continue,” they receive appropriate error messages or feedback. 

Ensure TalkBack users can easily understand and interact with the form. 

2. Make Fields Clearly Mandatory for All Users (Visual + Screen Reader) 

The radio buttons do not indicate that they are required. 

Ensure that mandatory fields are clearly indicated both visually and programmatically so that TalkBack announces them. 

3. Provide Feedback When User Skips the Mandatory Field 

If the user skips the required radio button and clicks "Continue," there’s no feedback. 

Implement validation that checks for required fields and provides feedback. When the user skips a required field, you need to ensure the form notifies the user with an error message. 

(Please note: The error alert approach implemented on the “Applicant Information” page can be reused in this section for consistency and accessibility.)
